core/src/main/java/io/ably/lib/transport/ConnectionManager.java

```diff
@@ -1447,6 +1429,26 @@ public QueuedMessage(ProtocolMessage msg, CompletionListener listener) {
         }
     }
 
+    /**
+     * Send all pending messages which are queue.
+     * Remove them from the queue once they are send successfully
+     * Spec: RTN19a
+     */
+    private void sendPendingQueueMessages() {
+        //RTN19a
+        if (pendingMessages.queue != null && !pendingMessages.queue.isEmpty()) {
+            for (final QueuedMessage queuedMessage : pendingMessages.queue) {
+                try {
+                    send(queuedMessage.msg, false, null);
+                } catch (AblyException e) {
+                    String errorString = String.format(Locale.ROOT, "Unable to send pending message %s (%s)",
```

Comment:
You can also emit errorString on `connection` as mentioned here => 
https://github.com/ably/ably-java/pull/842/files#r1014250157

Reply:
Can you just check ... I saw it somewhere sometimes bacl, it should be possible

Reply:
There is no way to emitUpdate() the same way as I did in the Presence.reEnter().
I am not sure if we should as every send event ends up being just logged.

Reply:
Please check this one -> https://github.com/ably/ably-java/blob/2775178075de882d6184b3ad46a8fa2707ec48f5/lib/src/main/java/io/ably/lib/transport/ConnectionManager.java#L1086

Reply:
@sacOO7 is https://github.com/ably/ably-java/pull/842/commits/865bca81e42e4f8665abcb847e21349d3f07fa42 ok?

Reply:
Looks okay. Just make sure, it's being sent for other places too